### DISCUSSION TITLE
How do I know the ID of the inserted value after I set auto_increase and insert data?

### DISCUSSION THREAD

**Discussion by victory-wu** [apache/doris#36573] [A - General / Q&A]
我设置了auto_increment , insert 数据后怎么知道插入的值的ID呢？

How do I know the ID of the inserted value after I set auto_increase and insert data?

**Comment by ixzc** [comment]
just select your auto_increment column

**Comment by shaotianfeng** [comment]
解决了吗 这个问题，我也遇到相同的问题 jdbc操作不了

Have you solved this problem? I also encountered the same problem. jdbc cannot operate it

**Reply by shaotianfeng** [reply]
jdbc insert不返回自增id

**Reply by victory-wu** [reply]
我理解是他不能返回ID，这个插入数据的过程是异步的。  估计后续可以通过API是去查询，这样干的话效率太低了。

直接用分布式ID得了
